### ISSUE TITLE
Docker Hub isn't up to date, apparently.

### ISSUE THREAD

**Issue by danwdart** [willhallonline/docker-ansible#22]
Docs say latest tracks alpine but it doesn't as it's 10 months old whereas alpine is 2 months old

**Comment by danwdart** [comment]
Hmm in fact python is still 2.7 in these... rather than 3 as in docs

**Comment by danwdart** [comment]
Oh I see what's happening - https://hub.docker.com/r/willhallonline/ansible/tags isn't up to date.

**Comment by willhallonline** [comment]
Hi @danwdart , actually we are both right and wrong. It seems that the automated builds did not work properly inside Docker Hub (I normally don't watch them as they take so long). So the containers have not been replaced with the Python 3 version 😢 .

There appears to have been a subtle change to the way that the automated builds are completed that maybe I misconfigured when doing the big change to Python3, which basically meant I removed all builds are replaced them. I will watch them today and try to make sure that they actually complete. Thanks for bringing to my attention. 👍 😃
